src/Backend/db/data.js

```diff
@@ -0,0 +1,167 @@
+const data=[
+    {
+        id:"1",
+        category:"shoes",
+        title:"Birk",
+        img: require('../../Assets/Shoes/birk.jpg'),
```

Comment:
Please avoid using `require()` to fetch the image. Use the source URL for the image instead.